composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "BSD-3-Clause",
     "require": {
         "php": ">=7.2",
-        "ramsey/uuid": "^3.8",
+        "ramsey/uuid": "^4.0",
```

Comment:
As @klaussilveira pointed out we can do >3.8.

Reply:
Changed to ^3.8|^4.0 as you suggested
![errocomposer](https://user-images.githubusercontent.com/73655563/164504149-2489191b-acfa-4f59-8a7b-1c4b9fd44358.png)